_posts/en/newsletters/2018-11-20-newsletter.md

```diff
@@ -0,0 +1,341 @@
+independent events is the square of the average.
+
+-->{% endcomment %}
+
+- **Slowed block production, increased fees:** as widely reported,
+  several miners and mining pools are producing blocks for competing
+  forks of Bitcoin Cash when they could likely earn greater revenues by
+  creating blocks for Bitcoin.  This is the likely cause of the
+  approximately 7% reduction in Bitcoin difficulty over the retarget
+  period ending Friday (UTC) and may mean additional decreases in
+  Bitcoin hash rate and difficulty for an unknown length of time.
+  Relevant consequences for Bitcoin businesses include:
+
+    - *Slower confirmation times:* the average time between blocks may
+      increase moderately to 11 or 12 minutes and the chance of there
+      being a long wait between blocks will increase significantly in
+      relative percentages (e.g. with historically typical 9 minute
+      average block intervals, about 0.7% of blocks take more than 45
+      minutes; with a 12 minute interval, 2.3% take more than 45
+      minutes).  Recommendation: Bitcoin users are already familiar with
+      occasional long delays, so likely no action is needed.
+
+    - *Possibly increased fees:* a longer time between finding blocks
+      means less space for transactions, which can cause fee increases.
+      Occasional long waits between blocks also tend to create sudden
+      fee spikes that can persist for hours afterwards.  Recommendation:
+      ensure your fee estimation is working correctly and consider
+      preparing any fee-reduction measures you're willing to use such as
+      payment batching.
+
+    - *Increased revenues for profit-maximizing miners:* each time
+      Bitcoin's difficulty adjusts downwards, mining becomes more
+      profitable for Bitcoin miners (all other things being equal).
+      Recommendation: do the math on reactivating slightly-old miners
+      and overclocking current miners.
+
+    - *Possible sudden end:* it's possible a large set of ideological miners
+      producing blocks for Bitcoin Cash will all return to mining for
+      the most profitable chain all at roughly the same time.  Combined
+      with any past difficulty decreases, this could produce a series of
+      Bitcoin blocks with shorter average time between blocks than
+      normal.  This will likely wipe out any moderate backlog and allow
+      fees to drop to their default minimums.  Recommendation: consider
+      preparing to perform fee-reducing [input consolidations][] if fees
+      drop to their minimums.
+
+- **Lightning protocol discussion:** over 75 emails have been posted to
+  the Lightning-Dev mailing list in the past week, representing almost
+  10% of the list's traffic in the past 365 days.  Many of the threads
+  continue conversations started at the protocol developers summit.  If
+  you're interested in Lightning protocol development, we suggest
+  reading each of this month's [threads][ln list].
```

Comment:
Was this supposed to link to a thread view of the LN mailing list? It links to the list info page.